src/client/assets/routes.js

```diff
@@ -20,12 +20,12 @@ const routes = [
 		displayText: 'Dashboard',
 		path: '/dashboard',
 	},
-	// {
-	// 	authLevel: [UserType.Hacker, UserType.Sponsor, UserType.Organizer],
-	// 	component: Profile,
-	// 	displayText: 'Profile',
-	// 	path: '/profile',
-	// },
+	{
+		authLevel: [UserType.Hacker, UserType.Sponsor, UserType.Organizer],
```

Comment:
We should really just delete commented out code from the codebase. It's a weird type of hoarding that just makes future developers nervous.

(CC: @aadibajpai )

Reply:
100% Agree